RevertConditions/TargetCall/README.md

```diff
@@ -0,0 +1,28 @@
+# Revert On Return Size Example
+
+## Overview
+This repository hosts a smart contract named `TargetCall`, featuring a function called `forward`. This function initiates the `execute` function of a specified target contract, set during the constructor invocation. The purpose of this example is to illustrate a revert condition based on the `TargetCall`. The compiler includes a check ensuring that the called target is a contract (verifies the code size). However, the prover can bypass this check by passing a regular address, resulting in a revert of the function.
```

Comment:
Perhaps clarify some things:
- "regular address _(i.e. one without code)_"
- "the prover can _bypass this check_" -> "the Prover can _create a violation_"
- "The compiler _includes_ a check" -> "The compiler _inserts_ a check"